content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: KIND
+description: Instructions to setup KIND for Istio.
+weight: 21
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/kind/
+    - /docs/setup/kubernetes/platform-setup/kind/
+keywords: [platform-setup,kubernetes,kind]
+---
+
+[KIND](https://kind.sigs.k8s.io/) is a tool for running local Kubernetes clusters using Docker container `nodes`.
+KIND was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+Follow these instructions to prepare KIND for Istio installation.
+
+## Prerequisites
+
+- Please use latest Go version, ideally go 1.13 or greater.
+- To use kind, you will also need to [install docker](https://docs.docker.com/install/).
+
+## Installation steps
+
+1.  Install the latest version of [KIND](https://kind.sigs.k8s.io/docs/user/quick-start/) using the command
```

Comment:
You have the link to the kind quick start (it's not really labelled as such), and then copied one of the options. I'd maybe suggest doing like you did for Docker above and just have them install via a link?

Reply:
I think it's good idea to install `kind` directly from there website. In this case we don't have to update our doc to reflect latest version of `kind`.